.github/local-development.md

```diff
@@ -0,0 +1,107 @@
+# Local Development
+
+This document gives tips and tricks on how to create the Renovate documentation website locally to add features or fix bugs.
+You can improve this documentation by opening a pull request.
+For example, if you think anything is unclear, or you think something needs to be added, open a pull request!
+
+## Installation
+
+### Prerequisites
+
+You need the following dependencies for local development:
+
+- Git
+- Node.js `^12.13.0 || >=14.15.0`
+- Yarn `^1.17.0`
+- Python `^3.8`
+- Bash shell with `make`
+
+We support Node.js versions according to the [Node.js release schedule](https://github.com/nodejs/Release#release-schedule).
+
+_Linux_
+
+You can use the following commands on Ubuntu.
+
+```sh
+curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -
+curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | sudo apt-key add -
+echo "deb https://dl.yarnpkg.com/debian/ stable main" | sudo tee /etc/apt/sources.list.d/yarn.list
+sudo apt-get update
+sudo apt-get install -y git python-minimal build-essential nodejs yarn
+```
+
+_Windows_
+
+QUESTION: Git Bash for Windows does not come with `make` by default, how should a user work around this? Or is Windows not supported for making the docs locally?
+
+Follow these steps to set up your development environment on Windows 10.
+If you already installed a component, skip the corresponding step.
+
+- Install [Git](https://git-scm.com/downloads). Make sure you've [configured your username and email](https://git-scm.com/book/en/v2/Getting-Started-First-Time-Git-Setup)
+- Install [Node.js LTS](https://nodejs.org/en/download/)
+- In an Administrator PowerShell prompt, run `npm install -global npm` and then `npm --add-python-to-path='true' --debug install --global windows-build-tools`
+- Install [Yarn](https://yarnpkg.com/lang/en/docs/install/#windows-stable)
+
+  Verify you're using the appropriate versions:
+
+  ```powershell
```

Comment:
I don't know if you like the syntax highlighting for the PowerShell terminal here or not, feel free to tell me to remove it again. 😄

Reply:
Ignore this comment, as this is no longer in the docs, we now point to WSL2 instead. 😄